src/App.tsx

```diff
@@ -1,25 +1,83 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import './CSS/App.css';
+import './CSS/nav-top.css';
+import './CSS/nav-middle.css';
+import './CSS/nav-bottom.css';
+import './CSS/bottom-bar.css';
+import './CSS/filter.css';
+import Header from "./Header";
+import Nav from "./Nav";
+import Footer from "./Footer";
+import Shop from "./PAGES/Shop";
+import Cart from "./PAGES/Cart";
+import SingleBeer from "./PAGES/SingleBeer";
+import Home from './PAGES/Home';
+import ConnectionError from './PAGES/ConnectionError';
+import Error from './PAGES/Error';
+import { BrowserRouter as Router ,Routes, Route, useNavigate } from "react-router-dom";
+import { createContext, useState } from "react";
+import getCartItemsSummary from './GETTERS/getCartItemsSummary';
+import beerType from './OBJECTS/beerType';
+
+export const filterKeyContext= createContext(Math.random())
 
 function App() {
+  
+  const [filterKey,setFilterKey] = useState(Math.random())
+  const filterUpdate = (): void => {
+    setFilterKey(Math.random())
+  }
+  const [cartSummary, setCartSummary] = useState(getCartItemsSummary())
```

Comment:
1. Research React apps to come up with a better directory structure
2. Rename directories to be lowercase
3. There's no reason for GETTERS dir to exist. GETTERS, OBJECTS names are too abstract. It's not clear what are we storing in them. If we want to create a helper class with local storage accessors we can create a LocalStorageService/Helper in services/helpers directory. If we want a classes to access data from API we can create e.g. BeerClient or BeerApi classes inside clients/api directory